balarco/utils.py

```diff
@@ -8,6 +8,19 @@
 from rest_framework.authtoken.models import Token
 
 
+STATUS = (
+    (0, 'Pendiente'),
+    (1, 'Diseño'),
+    (2, 'Cuentas'),
+    (3, 'Validación'),
+    (4, 'Producción'),
+    (5, 'Por cobrar'),
+    (6, 'Por facturar'),
+    (7, 'Pagado'),
+    (8, 'Terminado')
```

Comment:
I agree, this should be at least declared was constants, and this maybe used in other modules, maybe it would be useful to declare them in utils.py?